doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,166 @@ Release management
-* 1 meaning beta,
-* anything between 1 and 10 meaning stable, and
-* 10 meaning LTS.
+.. _release-diagram:
 
-So, each MAJOR release series goes through a development-maturity life cycle of
-MINOR releases, as follows:
+.. code-block::
 
-1. **Alpha**. Once in every few months we release a few alpha versions,
-   e.g. 2.0.1, 2.0.2.
 
-   Alpha versions may contain incompatible changes, crashes and other bugs.
+   1.10 series -- 1.10.4 -- 1.10.5 -- 1.10.6 -- 1.10.7
+   (LTS)
 
-2. **Beta**. Once major changes necessary to introduce new flagship features
-   are ready, we release a few beta versions, e.g. 2.1.3, 2.1.4.
+   ....
 
-   Beta versions may contain crashes, but do not have incompatible changes,
-   so can be used to develop new applications.
+   2.2 series --- 2.2.1 --- 2.2.2 --- 2.2.3 (end of support)
+                    |
+                    V
+   2.3 series ... 2.3.0 --- 2.3.1 --- 2.3.2 --- 2.3.3 (end of support)
+                              |
+                              V
+   2.4 series ............. 2.4.0 --- 2.4.1 --- 2.4.2
+                                        |
+                                        V
+   2.5 series ....................... 2.5.0 --- 2.5.1
+                                                  |
+                                                  V
+   2.6 series ................................. 2.6.0
 
-4. **Stable**. Finally, after we see our beta versions run successfully in
-   production, usually in a few more months, during which we fix all incoming
-   bugs and add some minor features, we declare this MAJOR release series
-   stable.
+   -----------------|---------|---------|---------|------> (time)
+                      1/4 yr.   1/4 yr.   1/4 yr.
 
-Like Ubuntu, we distinguish two kinds of stable releases:
+*Support* means that we continue fixing bugs. We add bug fixes simultaneously
+into the following release series: LTS, last stable, beta, and alpha.
+If we look at the release diagram above, it means that the bug fixes are to be
+added into 1.10, 2.4, 2.5, and 2.6 release series.
 
-* **LTS (Long Term Support)** releases that are supported for 3 years
-  (community) and up to 5 years (paying customers). **LTS** release
-  is identified by MINOR version 10.
-* **Standard stable releases** are only supported a few months after the next
-  stable is out.
+To sum it up, once a quarter we release (?see the release diagram above for
+reference):
 
-"Support" means that we continue fixing bugs in a release.
+* next LTS release, e.g., 1.10.7
+* two stable releases, e.g., 2.3.3 and 2.4.2
+* beta version of the next release series, e.g., 2.5.1.
 
-We add commits simultaneously to three MAJOR releases:
+In all supported releases, when we find and fix an outstanding CVE/vulnerability,
+we release a patch for that but do not tag a new PATCH level version.
+Users will be informed about such critical patches via official Tarantool news
+channel (`tarantool_news <https://t.me/tarantool_news>`_).
```

Comment:
> we release a patch <...> but do not tag a new PATCH level version.

Maybe it is not good to name the process of pushing a fix as 'release a patch'. It may be confusing in the context. I would think how to get rid of possible ambiguity.